optics-core/src/Optics/Internal/Optic/Subtyping.hs

```diff
@@ -31,7 +31,7 @@ instance {-# OVERLAPPABLE #-} TypeError ('ShowType k
                                          ':<>: 'Text " cannot be used as "
                                          ':<>: 'ShowType l
                                         ) => Is k l where
-  implies = error "unreachable"
+  implies _ _ = error "unreachable"
```

Comment:
So we'll **need** (at least a patch) release to support 8.10.1. Yet, I'm positively surprised so little broke